layers/meta-balena-beaglebone/recipes-core/images/balena-image.bbappend

```diff
@@ -6,3 +6,6 @@ IMAGE_ROOTFS_SIZE:beaglebone-ai64 = "733184"
 IMAGE_INSTALL:append:beaglebone = " bb-org-overlays fix-mmc-bbb bb-wl18xx-bluetooth bb-wl18xx-wlan0"
 IMAGE_INSTALL:append:beaglebone-ai64 = " mmc-utils vxd-dec-fw"
 IMAGE_INSTALL:append:beaglebone-pocket = " boot-scripts"
+IMAGE_INSTALL:append:beagleplay = " mmc-utils"
+
+IMAGE_ROOTFS_SIZE:beagleplay = "1966080"
```

Comment:
Root partitions are way too large, 733184 would be more appropriate

Reply:
@delhiryder

Reply:
Hi Alexandru, please refer to [this](https://balena-external.zulipchat.com/#narrow/stream/400911-Zylum/topic/AutoKit.20for.20BeaglePlay/near/444217197) thread from zulip. @floion and I have already gone down the path of trying to reduce the size of the root partition, but not had any success. Here's the original [post](https://balena-external.zulipchat.com/#narrow/stream/400911-Zylum/topic/BeaglePlay/near/394428372) from Alex (from last September).

Reply:
Hi @delhiryder , the links you shared take me to the extlinux discussions, I don't see anything about the boot and root partitions sizes. The problem with this is that the flasher image cannot be archived using zip due to the 4GB limit for zip files. cc @floion